File: C/Fibonacci.c
//Making it for n terms so that user can judje itself
#include <stdio.h>

int main(void) {
    int terms;
    int x = 0;
    int y = 1;
    int z = 0;
   // int terms = 10;     redeclaration
    printf("Enter the terms to find the fibonacci of");
    scanf("%d",&terms);
    for (int i = 0; i < terms; i++) {
        printf("%i ", x);
        z = x;
        x = y;
        y += z;
    }
    printf("\n");
    return 0;
}

File: Swift/Fibonacci.swift
import Foundation

func fibonacci(n: Int) -> Int {
  if n < 2 {
    return n;
  } else {
    return fibonacci(n: n - 1) + fibonacci(n: n - 2)
  }
}

for x in 0...10 {
    print(fibonacci(n:x))
}
